content/en/docs/setup/platform-setup/gardener/index.md

```diff
@@ -10,56 +10,56 @@ owner: istio/wg-environments-maintainers
+
+To learn more about the inception of this open source project, read [Gardener Project Update](https://kubernetes.io/blog/2019/12/02/gardener-project-update/), [Gardener - The Kubernetes Botanist](https://kubernetes.io/blog/2018/05/17/gardener/) on [`kubernetes.io`](https://kubernetes.io/blog).
 
-To set up your own [Gardener](https://gardener.cloud), see the
-[documentation](https://github.com/gardener/gardener/blob/master/docs/README.md)
-or have a look at the
-[landscape-setup-template](https://github.com/gardener/landscape-setup-template)
-project. To learn more about this open source project, read the
-[blog on `kubernetes.io`](https://kubernetes.io/blog/2018/05/17/gardener/).
+[Gardener yourself a Shoot with Istio, custom Domains, and Certificates](https://gardener.cloud/docs/extensions/others/gardener-extension-shoot-cert-service/docs/tutorial-custom-domain-with-istio/) is a detailed tutorial for the end user of Gardener.
 
 ### Install and configure `kubectl`
 
 1.  If you already have `kubectl` CLI, run `kubectl version --short` to check
-    the version. You need `v1.10` or newer. If your `kubectl` is older, follow the
+    the version. You need a current version that at least matches your Kubernetes
+    cluster version you want to order. If your `kubectl` is older, follow the
     next step to install a newer version.
 
-1.  [Install the `kubectl` CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+1.  [Install the `kubectl` CLI](https://kubernetes.io/docs/tasks/tools/).
 
 ### Access Gardener
 
 1.  Create a project in the Gardener dashboard. This will essentially create a
     Kubernetes namespace with the name `garden-<my-project>`.
 
-1.  [Configure access to your Gardener project](https://kubernetes.io/docs/tasks/tools/install-kubectl/#verifying-kubectl-configuration)
+1.  [Configure access to your Gardener project](https://gardener.cloud/docs/dashboard/usage/gardener-api/)
     using a kubeconfig. If you are not the Gardener Administrator already, you
     can create a technical user in the Gardener dashboard: go to the "Members"
     section and add a service account. You can then download the kubeconfig for
-    your project. You can skip this step if you create your cluster using the
-    user interface; it is only needed for programmatic access, make sure you set
-    `export KUBECONFIG=garden-my-project.yaml` in your shell.
+    your project. Make sure you set `export KUBECONFIG=garden-my-project.yaml` in your shell.
     ![Download kubeconfig for Gardener](images/gardener_service_account.png "downloading the kubeconfig using a service account")
+    You can skip this step if you intend to create your cluster using the
```

Comment:
Does it make sense to move the statement of optionally doing this step to the front of the step?

Reply:
thank you. embedded that statement in a <tip/>